src/inference/include/openvino/runtime/remote_tensor.hpp

```diff
@@ -43,7 +45,9 @@ class OPENVINO_RUNTIME_API RemoteTensor : public Tensor {
     template <typename T>
     T* data() = delete;
 
-    void copy_to(ov::Tensor& dst) const = delete;
+    void copy_to(ov::Tensor& dst) const;
+
+    void copy_from(ov::Tensor& src);
```

Comment:
do we need a reference here? `src` is not going to be changed

Reply:
Agree, replaced with `const ov::Tensor& src`